run.sh

```diff
@@ -1,8 +1,30 @@
 #!/bin/sh
 
+#add environmental variable of .env to the bash script
+. ./.env
+
+#Define absolute path for the host and the serv from the relative common path
+HOST_LOG_FILE=$(realpath -P ${LOG_FILE})
+
 PATH=$(/usr/bin/getconf PATH || /bin/kill $$)
 NAME="iscsc.fr-notify-bot"
 
+RED=$(tput setaf 1)
+NORMAL=$(tput sgr0)
+
+# Creating the file and setting the right permissions:
+
+# TODO: test $? and exit if ==1
+touch $HOST_LOG_FILE
+chmod 664 $HOST_LOG_FILE
+sudo chown :root $HOST_LOG_FILE
```

Comment:
you should execute this code only if `${HOST_LOG_FILE}` doesn't exists.
if it exists you should echo  a warning saying "take care that the log file have the right permissions" or something, and print its permissions (with `ls -l ${HOST_LOG_FILE}` for example)


Something like this could do well: (but test it and make your own research I'm not sure!)
```sh
if [ -f "${HOST_LOG_FILE}" ]; then
    echo "balbla"
else
    # the code
fi
```

Reply:
done in 09f8ed6

Reply:
tks!